Charging.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+
+#include "Charging.h"
+
+int chargingSamples[100];
+int NumOfChargingSamples;
+
+void BubbleSort(int chargingSamples[], int NumCS);
+void PrintSortedElements(int chargingSamples[], int NumCS);
+bool SampleAreConsecutive(int chargingSamples[], int NumCS);
```

Comment:
Naming could be slightly better - ex. AreSamplesConsecutive()

Reply:
Nomenclature improved